### PR TITLE
Ensure schema is still generated for empty dataframe

### PR DESCRIPTION
**Pull request by philippjfr**
Previously we were not using the dtype information to at least return a basic schema.

### CODE REVIEW

File: lumen/util.py [hunk 1]
```diff
@@ -277,10 +286,13 @@ def wrapper(*args, **kwargs):
             try:
                 return func(*args, **kwargs)
             except Exception as e:
-                log.error(
-                    f"{func.__qualname__!r} raised {type(e).__name__}: {e}"
-                )
-                pn.state.notifications.error(message.format(e=e))
+                if pn.state.notifications:
+                    log.error(
```
Comment (Hoxbro) [inline lumen/util.py:290]: Why does `log.error` depends on `pn.state.notification` here?

Comment (philippjfr) [reply lumen/util.py:290]: I prefer raising an actual error if we can't notify the user.

Comment (philippjfr) [reply lumen/util.py:290]: This would only really occur in a notebook/programmatic context where the user would actually see the error.

Comment (Hoxbro) [reply lumen/util.py:290]: Oh, read the else as part of the try/except and not the if block.

**Pull request merged**